css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+* {
+  font-family: sans-serif;
+  color: white;
+  font-weight: 300;
+}
+
+.background {
+  background-color: rgb(26, 150, 217);
+}
+
+.container {
+  max-width: 850px;
+  max-height: 600px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row-1 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.row-2 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
Rather than creating two different selectors and repeating yourself, just give these styles to your `.row`.